doc/md/language-manual.md

```diff
@@ -1209,14 +1209,14 @@ The declaration `<dec>` of a `system` field must be a manifest `func` declaratio
 | name          | type                                                          | description         |
 |---------------|---------------------------------------------------------------|---------------------|
 | `heartbeat`   | `() -> async ()`                                              | heartbeat action    |
-| `timer`       | `() -> async ()`                                              | timer action        |
+| `timer`       | `(Nat64 -> ()) -> async ()`                                   | timer action        |
```

Comment:
Shouldn't this be Nat64 -> Nat64 (or even Int64 to Int64 - does the API guarantee this is unsigned?)

Reply:
It is unsigned. But as Andriy pointed out, inside of the callback the timer is reset to 0, so there is no point in returning anything. This also makes it a pure setter, which is nicer.

Reply:
So I am testing this "being reset to 0" claim, but cannot confirm (at least on `drun`). `ic-ref` isn't tested yet. See https://github.com/dfinity/motoko/pull/3614.

Reply:
Looks like, indeed, the most recent versions of `ic-ref` and `drun` reset the global timer to 0 before entering the expiration callback.